src/js/translate-visitor.ts

```diff
@@ -19,9 +19,13 @@ export default class TranslateVisitor extends types.PathVisitor implements  type
     };
     currentContext: types.Context;
 
-    constructor(private loader: TranslateLoaderContext) {
+    constructor(private loader: TranslateLoaderContext, options: any) {
         super();
 
+        options = options || {};
```

Comment:
The current version of typescript/typings is causing issues with Object.assign and object rest spread, so I ended up using an npm package which does the same thing [object-assign](https://www.npmjs.com/package/object-assign)